app/build.gradle

```diff
@@ -7,8 +7,8 @@ android {
     defaultConfig {
         minSdkVersion 17
         targetSdkVersion 23
-        versionCode 2
-        versionName "2.0"
+        versionCode 3
```

Comment:
https://developer.android.com/studio/publish/versioning.html Here you can find more details on the topic.
Long things short: _versionCode_ is a number just to make version comparison easier (it should be always greater for newer version)  and versionName could be any string. The format is not strictly limited there.